documentation/user_docs/docs/user_guide/Getting_started.rst

```diff
@@ -482,85 +655,201 @@ This should give a plot that looks like this:
-different from those of ``demo_4gauss`` - to see the differences it is probably easiest to examine the code for the two
-functions side-by-side.
+.. note::
+
+   The difference between ``sqw_eval`` and ``func_eval`` is the way in which the
+   arguments are passed to the target function. ``sqw_eval`` passes the
+   positions (h,k,l,e) for each point, while ``func_eval`` simply passes the
+   calculated intensity.
+
+The function we will use to demonstrate here is a model appropriate for spin
+excitations of a 3D Heisenberg ferromagnet; it is called
+``FM_spinwaves_2dSlice_sqw``, and it takes as inputs the components of **Q**
+(h,k,l) plus energy, as well as other function parameters (exchange constant
+etc.).
+
+Please take a look at the code of the example function by typing:
 
 ::
 
-   w_sim= sqw_eval (w_template,@FM_spinwaves_2dSlice_sqw,[300 0 2 10 2]);
+   edit FM_spinwave_2dSlice_sqw
+
+into the Matlab terminal.
+
+You should notice that the format of the inputs for this function are thus
+different from those of ``demo_4gauss`` -- to see the differences it is easiest
+to examine the code for the two functions side-by-side.
+
+Tun call the function we run:
+
+::
+
+   w_sim = sqw_eval(w_template, @FM_spinwaves_2dSlice_sqw, [300 0 2 10 2]);
+
+The similar syntax makes it easier to remember how to apply general functions,
+while the ``sqw_eval`` vs. ``func_eval`` allows us to define specialised and
+flexible functions for computing transformations of our data.
+
+.. note::
+
+   In general it is better to use ``func_eval`` for simple functions such as
+   Gaussians and so on, and ``sqw_eval`` for "proper" models of the scattering.
+
 
+.. note::
 
-In general it is better to use ``func_eval`` for simple functions such as Gaussians and so on, and sqw for "proper"
-models of the scattering. The different syntax makes it easier to keep track of what kind of model for the scattering is
-being employed. As before, the keyword ``'all'`` can be added to the arguments of this function, however in this case it is
-ignored if the object ``w_template`` is an sqw object. If ``w_template`` is a dnd object then as for func_eval the
-keyword ``'all'`` ensures that data are simulated over the entire data range. As with ``func_eval``, the parameters passed
-to the function can either take the form of a vector of numerical parameters, or a cell array comprising any other form
-of input.
+   As before, the keyword ``'all'`` can be added to the arguments of this
+   function, however in this case it is ignored if the object ``w_template`` is
+   an ``sqw`` object.
+
+   If ``w_template`` is a ``dnd`` object then, as for ``func_eval``, the keyword
+   ``'all'`` ensures that data are simulated over the entire data range.
+
+.. note::
+
+   As with ``func_eval``, the parameters passed to the function can
+   either take the form of a vector of numerical parameters, or a cell array
+   comprising any other form of input.
+
+
+Bose Temperature
```

Comment:
You say that this section is about apply so maybe have that as the section heading